wicket/src/cli/inventory.rs

```diff
@@ -0,0 +1,123 @@
+                // something different here.
+                // - We want the status of only sleds we've configured wicket
+                //   to use for setup. /bootstrap-sleds will give us sleds
+                //   we don't want
+                // - We want the status even if they aren't visible on the
+                //   bootstrap network yet.
+                //
+                // In other words, we want the sled information displayed at the
+                // bottom of the rack setup screen in the TUI.
+                let conf = client
+                    .get_rss_config()
+                    .await
+                    .context("failed to get rss config")?;
+
+                let bootstrap_sleds = &conf.insensitive.bootstrap_sleds;
+                let sled_data: Vec<ConfiguredBootstrapSledData> =
+                    bootstrap_sleds
+                        .iter()
+                        .map(|desc| {
+                            let slot = desc.id.slot;
+
+                            let identifier = match &desc.baseboard {
+                                Baseboard::Gimlet { identifier, .. } => {
+                                    identifier.clone()
+                                }
+                                Baseboard::Pc { identifier, .. } => {
+                                    identifier.clone()
+                                }
+                                Baseboard::Unknown => "unknown".to_string(),
+                            };
+
+                            let address = desc.bootstrap_ip;
+
+                            ConfiguredBootstrapSledData {
+                                slot,
+                                identifier,
+                                address,
+                            }
+                        })
+                        .collect();
+
+                if json {
+                    let json_str = serde_json::to_string(&sled_data)
+                        .context("serializing sled data failed")?;
+                    println!("{}", json_str);
+                } else {
+                    for sled in &sled_data {
+                        print_bootstrap_sled_data(sled);
+                    }
+                }
+
+                Ok(())
+            }
+        }
+    }
+}
+
+#[derive(Serialize)]
+struct ConfiguredBootstrapSledData {
+    slot: u32,
+    identifier: String,
+    address: Option<Ipv6Addr>,
+}
+
+fn print_bootstrap_sled_data(data: &ConfiguredBootstrapSledData) {
+    let ConfiguredBootstrapSledData { slot, identifier, address } = data;
+
+    // Print status indicator
+    let status = match address {
+        None => format!("{}", '⚠'.red()),
+        Some(_) => format!("{}", '✔'.green()),
+    };
+
+    let addr_fmt = match address {
+        None => "(not available)".to_string(),
+        Some(addr) => format!("\t{}", addr),
+    };
+
+    // The rest of the data
+    println!("{status} Cubby {:02}\t{identifier}{addr_fmt}", slot);
```

Comment:
@sunshowers i saw in the other comment above you asked for the field to be changed from `slot` to `cubby`, so:

* is there a distinction between what "slot" vs "cubby" describes? i've seen "cubby" elsewhere and i'm kinda wondering if it's an (implicit?) word choice change or something else.
* should this be `{status} Slot {:02}`? :grin:

Reply:
yeah i am also not sure about that. This says Cubby because the wicket TUI also says Cubby today and I am just matching what that does.

Reply:
> is there a distinction between what "slot" vs "cubby" describes? i've seen "cubby" elsewhere and i'm kinda wondering if it's an (implicit?) word choice change or something else.

I think we've used the two terms interchangeably, not aware of specific differences between the two -- I know that the data structures internally mostly use "slot" which is why I was hoping at least the JSON could use "slot".